dronegen/publish.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+const relcliImage = "146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/relcli:v1.1.70-beta.3"
+
+func publishReleasePipeline() pipeline {
+	p := newKubePipeline("publish-rlz")
```

Comment:
You might want to consider creating a function that sets up a "rlz pipeline" with the appropriate env variables and pull step. This would reduce the repetition between `publish-rlz` and `clean-up-previous-build`